src/content-distribution/incoming-post/style.scss

```diff
@@ -28,19 +28,16 @@
 	}
 
 	.editor-sidebar:not(#newspack-network-incoming-post\:newspack-network-content-distribution-panel, #edit-post\:document) {
+		@include grey-out-outer;
 
```

Comment:
Looks like this one might also overwrite  #193

Reply:
🤦  I did a very bad job fixing conflicts, sorry about that!

Fixed in 8ed2220456cc0e3c29bd5a60c68c7bb063684355